### DISCUSSION TITLE
Using with SecretsManager driver

### DISCUSSION THREAD

**Discussion by maciejwalkowiak** [awslabs/aws-postgresql-jdbc#41] [Q&A]
Some time ago there was a very promising project - https://github.com/aws/aws-secretsmanager-jdbc/ - not sure if its still under development or is it deprecated. Is there a plan to have similar support for Secrets Manager in `aws-postgresql-jdbc` driver? Combining AZ failover with authentication via Secrets Manager in single driver seems reasonable to me.

**Comment by adalevinAWS** [comment]
Thank you for the suggestion. We will evaluate this for the roadmap.

**Comment by hsuamz** [comment]
For visibility, we are recommending everyone to move to our new repository here: https://github.com/awslabs/aws-advanced-jdbc-wrapper.

We have redesigned the driver as a JDBC Wrapper which has been tested to work with the PGJDBC driver. Please give it a look and we will be continuing discussion there.

Secrets Manager is now supported.

Thanks.
